lib/Minz/Request.php

```diff
@@ -274,6 +311,20 @@ public static function forward($url = array(), $redirect = false) {
 		}
 
 		$url = Minz_Url::checkUrl($url);
+		$requestId = self::requestId();
+		$url['params']['rid'] = $requestId;
+
+		//Forward notification
+		$notif = self::getNotification();
+		if ($notif) {
```

Comment:
Could alternatively be moved to `setNotification()`

Reply:
Well, yes :+1: I don't understand why we would save the notification both in session `notification` AND `requests` parameters. It looks to me we can replace the code of `setNotification()`  by the following lines.

Reply:
OK, here you are https://github.com/FreshRSS/FreshRSS/pull/3208/commits/76dc983942980044b53807b227511edf8d2c0d5d